Makefile.Common

```diff
@@ -54,7 +54,7 @@ $(TOOLS_BIN_DIR):
 	mkdir -p $@
 
 $(TOOLS_BIN_NAMES): $(TOOLS_BIN_DIR) $(TOOLS_MOD_DIR)/go.mod
-	cd $(TOOLS_MOD_DIR) && $(GOCMD) build -o $@ -trimpath $(filter %/$(notdir $@),$(TOOLS_PKG_NAMES))
+	cd $(TOOLS_MOD_DIR) && GOOS="" $(GOCMD) build -o $@ -trimpath $(filter %/$(notdir $@),$(TOOLS_PKG_NAMES))
```

Comment:
It seems reasonable to enforce this here since these tools are be used locally - at first I can't see a typical case in which we want this to follow `GOOS`. This is also showing that the step to check the cache, used to skip `make install-tools` is also wrong.